web/themes/custom/votegov/templates/navigation/menu--main--votegov-mainnavigation.html.twig

```diff
@@ -49,7 +49,7 @@ We call a macro which calls itself to render the full tree.
       <ul class="usa-nav__primary usa-accordion" data-test="mobileNav">
         {% for item in items %}
           {% set aria_id = aria_id ~ loop.index | clean_id %}
-          <li class="usa-nav__primary-item" aria-current="{{ item.in_active_trail ? 'page' : 'false' }}">
+          <li class="usa-nav__primary-item">
```

Comment:
after doing more reading i found that the `aria-current="false"` is not a needed label it is assumed that if it does not have the `aria-current="page"` then it is good